app/src/main/java/org/schabi/newpipe/report/ErrorActivity.java

```diff
@@ -377,8 +378,12 @@ private String getUserActionString(UserAction userAction) {
     }
 
     private String getContentLangString() {
```

Comment:
Three questions:
1. Why is **getContentLangString** giving **content country**?
2. Shouldn't we build a localization and return its toString, with both content Language and content country
3. Should I use Locale.getDefault().toString() or another one from them (thanks [stackoverflow](https://stackoverflow.com/a/23168383))
```
Locale.getDefault().getLanguage()       ---> en      
Locale.getDefault().getISO3Language()   ---> eng 
Locale.getDefault().getCountry()        ---> US 
Locale.getDefault().getISO3Country()    ---> USA 
Locale.getDefault().getDisplayCountry() ---> United States 
Locale.getDefault().getDisplayName()    ---> English (United States) 
Locale.getDefault().toString()          ---> en_US
Locale.getDefault().getDisplayLanguage()---> English
Locale.getDefault().toLanguageTag()     ---> en-US
```

Reply:
1. That's a mistake. I already fixed it while working on https://github.com/TeamNewPipe/NewPipe/issues/1816. However, I didn't push the changes. I'll rebase that branch and push it an Friday or Saturday. There was one thing which kept me from creating a PR though: I added extended debugging which allows one to get the documents which could not be parsed and attend them to the crash report. As a result the crash reports could easily be 1BM and more. Copying and pasting them into the a GitHub app or the browser caused serious lagging on my phone. So it might be better to store the documents in a .zip file and ask the user to upload it. I didn't have time to implement this.
2. I added a content country field, too. I guess the best thing is to only print the codes which are stored in the app settings.
3. Should be this. `Locale.getDefault().toString()          ---> en_US`

Reply:
I already put toString so it's ok. Is the array initialization good now?